### DISCUSSION TITLE
What is the best way to display author email or author website in a Quarto book?

### DISCUSSION THREAD

**Discussion by ferenci-tamas** [quarto-dev/quarto-cli#11383] [Q&A] [answered]
### Description

I have no idea how to display author email or author website in a Quarto book. I have found two approaches, but neither seems to be satisfactory:

1) Using the following solution:

```
author:
  - name: My Name
  - url: mailto:email@email.edu
```
This simply does not work for a book, where I should add something to the `_quarto.yml` (probably it is OK for a single document).

2) Using `My Name^[<email@email.edu>]` in `book` / `author` within the `_quarto.yml`, which is not really elegant (it does not respect the semantics, as gives information other than author in the author field) and also produces a very strange result (the footnote is repeated four times!).

**Comment by mcanouil** [comment, marked as answer]
Why aren't you following the author schema? Any reasons?

- https://quarto.org/docs/journals/authors.html

**Reply by ferenci-tamas** [reply]
Thanks! I thought I tried this approach (to no avail), but now I see that what I actually tried was the approach 1 given in my question, which is indeed not working, but because of a completely different error (there was an unnecessary hyphen before `url`).

So thank you very much, your solution is working perfectly. I just have one last question: is it possible to somehow include these in the PDF as well? None of this seems to appear there (except for the name itself).

**Reply by mcanouil** [reply]
Yes, either use a template made by the community or modify the partial/templates of the format you are targeting.

- https://quarto.org/docs/journals/templates.html#overview
- https://m.canouil.dev/quarto-extensions/

**Reply by ferenci-tamas** [reply]
Thank you very much!

Shouldn't it be opened as a feature request for Quarto? I thought it'd be natural to include these in a PDF by default (so that it does not require manual modification of the template).

**Reply by mcanouil** [reply]
The templates mostly comes from Pandoc.

Template extensions are there to allow whatever you want. For instance, there are several Scientific journal templates.

>  I thought it'd be natural to include these in a PDF by default 

That's very subjective. And as I said, you can make your own small Quarto extension that does add this.

Take a look at the following Quarto extension for example: https://github.com/nmfs-opensci/quarto_titlepages

**Reply by ferenci-tamas** [reply]
> That's very subjective.

OK, I see your point. Thank you once again!
